make/readline.mk

```diff
@@ -109,6 +109,9 @@ readline-source: $(DL_DIR)/$(READLINE_SOURCE) $(READLINE_PATCHES)
 # If the compilation of the package requires other packages to be staged
 # first, then do that first (e.g. "$(MAKE) <bar>-stage <baz>-stage").
 #
+
+export LD_LIBRARY_PATH=/home/jenkins/workspace/ManagementTools/Optware_Package/staging/opt/lib
```

Comment:
Ideally this path should be relative to the build workspace which current is /home/jenkins/workspace/ManagementTools/Optware_Package but this could change. Try running this with a relative path of ./staging/opt/lib, if that doesn't work maybe try using pwd to get the current path to provide a full path.

Reply:
Updated to use $(STAGING_DIR).